### PR TITLE
[DOC]: Docker compose warning

### PR DESCRIPTION
**Pull request by tazarov**
- Added warning to all docker compose commands to indicate ephemeral data with instructions on how to fix that

### CODE REVIEW

**Comment by tazarov** [comment]
Ref: https://discord.com/channels/1073293645303795742/1149400313997049936

**Comment by tazarov** [comment]
This is related to: https://github.com/chroma-core/chroma/issues/1119

File: docs/getting-started.md [hunk 1]
```diff
@@ -58,7 +58,32 @@ const {ChromaClient} = require('chromadb');
 const client = new ChromaClient();
 ```
 
-To connect to Chroma's backend - you either need to connect to a hosted version of Chroma, or run it on your local computer. If you can run `docker-compose up -d --build` you can run Chroma. 
+To connect to Chroma's backend - you either need to connect to a hosted version of Chroma, or run it on your local computer. If you can run `docker-compose up -d --build` you can run Chroma.
+
+:::caution Ephemeral Storage 💀
```
Comment (jeffchuber) [inline docs/getting-started.md:63]: can we simplify this whole section? I think either it should fit in one small callout, or we should find a different place to put it other than `getting started`

**Review by jeffchuber — CHANGES_REQUESTED** [review]

File: docs/getting-started.md (hunk 1 above)
Comment (tazarov) [reply docs/getting-started.md:63]: We can add it to the Troubleshooting section and just link to it.

```
:::caution Ephemeral Storage 💀
By default docker compose will store your data in a local volume see <link to troubleshooting>
:::
```

Or maybe we add a new Tips&Tricks/FAQ section where to put such tidbits, wdyt?

**Comment by jeffchuber** [comment]
closing for staleness - discussed outside of github

**Pull request closed without merging**